services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -40,6 +40,19 @@ class ProjectFileService(
       }
     }
 
+  def getFile(
+    projectId: ProjectId,
+    path: Path,
+    version: Option[PipelineVersion] = None,
+    userId: UserId
+  ): Future[ProjectFile] =
+    projectService.getUserProjectById(projectId, userId).flatMap { project =>
+      projectVersioning.getFile(project, path, version).flatMap {
+        case Left(versioningException) => Future.failed(versioningException)
+        case Right(projectFile)        => Future.successful(projectFile)
+      }
+    }
```

Comment:
We should add tests for this method too

Reply:
Yes, I really forgot to do it. Thank you for reminding.!

Reply:
I wrote 3 additional tests